tools/test_api.py

```diff
@@ -1647,6 +1647,12 @@ def get_module_avail(module_name):
     """
     return module_name in sys.modules.keys()
 
+def get_mounted_details_txt(mount_point):
```

Comment:
Why is this function in `test_api.py`? I don't think it logically fits here.

Reply:
@theotherjimmy Happy to move it- my original thought process was to add it into test_api.get_autodetected_MUTS_list and return it as a mut object. After realising it would break the mut .json format, I decided to put it above. Any suggestions on where it may fit in better?

Reply:
Since it's only used in `detect_targets.py` let's put it there.